src/AlertBox/AlertBox.jsx

```diff
@@ -0,0 +1,70 @@
+import _ from "lodash";
+import classnames from "classnames";
+import React, {PureComponent, PropTypes} from "react";
+
+import "./AlertBox.less";
+import {CloseIcon, WarningIcon, SuccessIcon, ErrorIcon, InfoIcon} from "./Icons";
+
+/*
+ * AlertBox is a closable, highlighted box
+*/
+export default class AlertBox extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {isOpen: true};
+  }
+
+  closeBox() {
+    this.setState({isOpen: false});
+  }
+
+  render() {
+    const {children, className, type, title, isClosable} = this.props;
+    const {isOpen} = this.state;
+    const {Icons, cssClass} = AlertBox;
+    if (!isOpen) {
+      return null;
+    }
+    return (
+      <div className={classnames(`AlertBox--${type}`, cssClass.CONTAINER, className)}>
+        <div className={cssClass.HEADER}>
+          <span>{Icons[type]()}</span>
+          <strong className={cssClass.TITLE}> {title} </strong>
+          {isClosable &&
+            <button className={cssClass.CLOSE} onClick={() => this.closeBox()}><CloseIcon /></button>
+          }
+        </div>
+        {children}
+      </div>
+    );
+  }
+}
+
+AlertBox.Type = {
```

Comment:
No need for `Type` to be uppercased, maybe call this `AlertBox.types` or capitalize it entirely, and call it `TYPES`. Additionally it doesn't need to be a member of `AlertBox`, actually. So how about just

```
const TYPES = {
  //...
};
```

If you want them to be accessible outside of `AlertBox.jsx`, then you can `export` this as a named export as well.